src/Controller/StoreApiController.php

```diff
@@ -24,6 +24,7 @@
 
 namespace Adyen\Shopware\Controller;
 
+use Adyen\AdyenException;
```

Comment:
To remove

Reply:
https://github.com/Adyen/adyen-shopware6/pull/195/commits/67e794880d23031e235b428ec19690caa1053202